.github/workflows/ci.yml

```diff
@@ -67,4 +67,47 @@ jobs:
           DATABASE_URI: "postgresql://postgres:postgres@postgres:5432/testdb"
 
       - name: Upload code coverage
-        uses: codecov/codecov-action@v3.1.4
\ No newline at end of file
+        uses: codecov/codecov-action@v3.1.4
+  bdd-tests:
+    runs-on: ubuntu-latest
+    container: rofrano/pipeline-selenium:latest
+    services:
+      postgres:
+        image: postgres
+        env:
+            POSTGRES_PASSWORD: postgres
+            POSTGRES_DB: testdb
+        ports:
+            - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+            --health-cmd pg_isready
+            --health-interval 10s
+            --health-timeout 5s
+            --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install OS dependencies
+        run: |
+            apt-get update
+            apt-get install -y gcc libpq-dev
+
+      - name: Install Python packages
+        run: |
+            python -m pip install --upgrade pip wheel
+            pip install -r requirements.txt
+
+      - name: Run the service locally
+        run: |
+            echo "\n*** STARTING APPLICATION ***\n"
+            gunicorn --log-level=critical --bind=0.0.0.0:8080 service:app &
```

Comment:
Hi Jay, could you clarify where / why we would run honcho start over the current implementation?

Reply:
Hey @studentgrad0419, I just mean that in our project, when we run honcho start, to run our app it runs on 8000.  I'm not sure if we will run into problems here or not.  I suppose if everywhere in the container the $PORT is set to 8080 then it should work.  But up to this point we've been running it on 8000.

Reply:
@jay-goettelmann-nyu I understand the concern, but from what I could tell it seems to be working.
I think this is because we are running a second job in a different container in the automated tests.
That means port 8080 should be free if not in use by anything else.

See screenshot / checks on this PR to show that behave is working.
![image](https://github.com/CSCI-GA-2820-FA23-003/wishlists/assets/112780372/385bafad-ba2f-4df9-aa6a-8165beb04a3b)